index.html

```diff
@@ -0,0 +1,10 @@
+<html>
+<head>
```

Comment:
w3c: **Error**: Element head is missing a required instance of child element title.